tox.ini

```diff
@@ -103,6 +103,9 @@ deps =
   # -c{toxinidir}/docs/requirements.txt
 description = Build The Docs
 changedir = {toxinidir}/docs
+# Reinstall so setuptools-scm runs and updates the _version.py
+# intentionally not in alphabetical order due to execution order
+commands_pre = {envpython} -m pip install --no-deps -e {toxinidir}
```

Comment:
Also, given that the current implementation does not use the dist metadata, this isn't necessary, strictly speaking.

Reply:
True, but the tox env not being up to date seems odd, what I really should add is `git clean -iX -- docs/pkg/` because I've been bitten by that with failed tox docs builds a bunch

Reply:
True, that is a great candidate for the next PR :)